core/core.iml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please rebase you fork. Common project contain now .gitignore files with rule ignone *.iml